templates/CMakeLists.txt.template

```diff
@@ -482,6 +487,19 @@
   %   endif
   % endfor
 
+  if(gRPC_BUILD_TESTS)
```

Comment:
You definitely should not need this. You should be relying on the target you identified from libuv.cmake (which is oddly named libuv_a)

Reply:
Thanks for pointing this out! I removed this part and the builds are still passing.